lib/sink.js

```diff
@@ -66,6 +66,22 @@ module.exports = class SinkMem extends EventEmitter {
         this.db = {};
     }
 
+    async get(filename) {
+        return this.db[filename] ? this.db[filename] : null;
```

Comment:
is null the same response from the gcs sink?

Reply:
I guess we should reject....?